doc/release-notes/2023-04-05.rst

```diff
@@ -0,0 +1,88 @@
+.. _release_notes_2023_04_05:
+
+Highlights from 2023-04-05 release
+==================================
+
+Updated client packages are shipped in `Fedora 36`_,  `Fedora 37`_,  `Fedora 38`_
+`EPEL 7`_, `EPEL 8`_, and `EPEL 9`_ Bodhi updates.
+
+
+Disable AppStream for existing Copr projects
+--------------------------------------------
+
+We decided to disable AppStream metadata generation for all project
+excluding those mentioned in `Fedora discussion`_ due to very high I/O
+demand of ``appstream-builder`` tool which causes failures in builds
+even for medium-sized projects. If you want to enable generating AppStream
+data for your project, please go to project settings and enable it back.
+(`#2577`_)
+
+
+Fixing slow searching in Fedora Copr projects
+---------------------------------------------
+
+Searching in Copr was slow and often timeout. We discussed options
+to switch to another search engine since our current search engine
+``whoosh`` seemed to be slow. We managed to speed up the search speed
+by a ton. (`#2534`_)
+
+
+Notable changes
+---------------
+
+- We are checking whether the user can build in the project before
+  uploading SRPM starts. (`#2524`_)
```

Comment:
It is better to form it in way "previously ... what was happening ... and now.... how we fixed it".
So this sentence: Previously, when you uploaded src.rpm to be built in non-enabled chroot, Copr emitted an error after the upload of src.rpm. Now we check the chroot before you upload the src.rpm.